pkg/scheduler/core/generic_scheduler.go

```diff
@@ -157,28 +156,11 @@ func (g *genericScheduler) Schedule(ctx context.Context, prof *profile.Profile,
 		}
 	}
 
-	metrics.DeprecatedSchedulingAlgorithmPredicateEvaluationSecondsDuration.Observe(metrics.SinceInSeconds(startPredicateEvalTime))
-	metrics.DeprecatedSchedulingDuration.WithLabelValues(metrics.PredicateEvaluation).Observe(metrics.SinceInSeconds(startPredicateEvalTime))
-
-	startPriorityEvalTime := time.Now()
-	// When only one node after predicate, just use it.
-	if len(feasibleNodes) == 1 {
-		metrics.DeprecatedSchedulingAlgorithmPriorityEvaluationSecondsDuration.Observe(metrics.SinceInSeconds(startPriorityEvalTime))
-		return ScheduleResult{
```

Comment:
this shouldn't be removed -- the scheduler should still return immediately when there's only one feasible node

Reply:
Thanks for catching that..fixed it